Scenarios/Contribute.md

```diff
@@ -3,31 +3,32 @@
 This section describes how to contribute to AL-Go. How to set up your own environment (your own set of actions and your own templates)
 
 You can do this in two ways:
+
 - Use a fork of AL-Go for GitHub in your own **personal GitHub account** in development mode
 - Use 3 public repositories in your own **personal GitHub account** (AL-Go-PTE, AL-Go-AppSource and AL-Go-Actions, much like in production)
 
 ## Use a fork of AL-Go for GitHub in "development mode"
 
 1. Fork the [https://github.com/microsoft/AL-Go](https://github.com/microsoft/AL-Go) repository to your **personal GitHub account**.
-2. You can optionally also create a branch in the AL-Go fork for the feature you are working on.
+1. You can optionally also create a branch in the AL-Go fork for the feature you are working on.
 
 **https://github.com/<yourGitHubUserName>/AL-Go@<yourBranch>** can now be used as a template in your AL-Go project when running _Update AL-Go System Files_ to use the actions/workflows from this fork.
 
 ## Use 3 public repositories in "production mode"
 
 1. Fork the [https://github.com/microsoft/AL-Go](https://github.com/microsoft/AL-Go) repository to your **personal GitHub account**.
-2. Navigate to [https://github.com/settings/tokens/new](https://github.com/settings/tokens/new) and create a new personal access token with **Full control of private repositories** and **workflow** permissions.
-3. In your personal fork of AL-Go, create a New Repository Secret called **OrgPAT** with your personal access token as content. See **https://github.com/yourGitHubUserName/AL-Go/settings/secrets/actions**.
-4. In your personal fork of AL-Go, navigate to **Actions**, select the **Deploy** workflow and choose **Run Workflow**.
-5. Using the default settings press **Run workflow**. Select the AL-Go branch to run from and the branch to deploy to.
+1. Navigate to [https://github.com/settings/tokens/new](https://github.com/settings/tokens/new) and create a new personal access token with **Full control of private repositories** and **workflow** permissions.
+1. In your personal fork of AL-Go, create a New Repository Secret called **OrgPAT** with your personal access token as content. See **https://github.com/yourGitHubUserName/AL-Go/settings/secrets/actions**.
```

Comment:
Should we include some information about how to setup pre-commit in this section?